### DISCUSSION TITLE
Unhandled Promise rejection: __webpack_require__.nmd is not a function

### DISCUSSION THREAD

**Discussion by ronakj** [webpack/webpack#17749] [Q&A] [answered]
Hi,

I have been facing this issue while trying to upgrade from webpack 4 to 5 for a package. I can't share a repro as the code is internal and huge and I don't think I will be able to create a small repro without details.

I can share some of the configuration which I feel might be relevant -

webpack config 1 - for `override bundle`
```js
  output: {
     ...
     file: "[name].js",
     libraryType: "umd"
  },
  ...
  plugins: [
      new webpack.optimize.LimitChunkCountPlugin({
        maxChunks: 1,
      })
    ],
  ...
  externals: ["react", "react-dom", ...]
```

webpack config 2
```js
  output: {
     ...
     libraryType: "umd"
  },
  alias: {
     override:  "/output/of/config/1"
   },
  externals: ["react", "react-dom", ...]
```

The output of webpack config 2 is loaded through a host app which we don't have control over. The override bundle is used for loading a override bundle though config 2 instead of the production bundle for dev testing. I am seeing this issue when loading the override bundle.

![image](https://github.com/webpack/webpack/assets/2021979/4d91749e-500a-45a4-8df5-d940f094c0db)

I haven't really changed anything in the output options, or external options when migrating from webpack 4 to 5. I only updated a few plugin versions and changed a few config options which were renamed.

I know this is not much information to go on, but if anyone has an idea why this error can occur please let me know.

**Comment by vrrj17** [comment]
I have the similar issue facing and it is pointing one of the lodash module
<img width="481" alt="Screen Shot 2023-12-08 at 5 59 26 PM" src="https://github.com/webpack/webpack/assets/92488609/4c9062d6-07e8-46c9-8288-821bdb57cd55">

**Comment by alexander-akait** [comment]
Please create reproducible test repo, thank you

**Comment by ronakj** [comment, marked as answer]
Don't have one. Issue was most likely due to some externals not being present I think, it got fixed anyhow.
